src/Transport/Receiving/RepeatedFailuresOverTimeCircuitBreaker.cs

```diff
@@ -7,12 +7,11 @@
 
     class RepeatedFailuresOverTimeCircuitBreaker
     {
-        public RepeatedFailuresOverTimeCircuitBreaker(string name, TimeSpan timeToWaitBeforeTriggering, Action<string, Exception, CancellationToken> criticalError, CancellationToken messageProcessingCancellationToken)
+        public RepeatedFailuresOverTimeCircuitBreaker(string name, TimeSpan timeToWaitBeforeTriggering, Action<Exception> triggerAction)
```

Comment:
Following the pattern of the [equivalent type in MSMQ](https://github.com/Particular/NServiceBus.Transport.Msmq/blob/1c543cad78842a735c0c275246733ad893117e9e/src/NServiceBus.Transport.Msmq/RepeatedFailuresOverTimeCircuitBreaker.cs#L10).